CONTRIBUTING.md

```diff
@@ -11,6 +11,15 @@ expectations the Node Foundation requires of its contributors.
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Collaborator** is a subset of contributors who have been given write access to the repository.
 
+## Start Here:
+
+- Take a look at any Issues tagged with [Good First Contribution](https://github.com/nodejs/community-committee/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+contribution%22)
```

Comment:
Add a period at the end. Any reason Issues is capitalized? It doesn't need to be.